frontend/packages/dev-console/src/components/topology/list-view/TopologyListView.tsx

```diff
@@ -105,6 +105,17 @@ const ConnectedTopologyListView: React.FC<TopologyListViewProps &
       (n) => n.getType() !== TYPE_APPLICATION_GROUP && isGraph(n.getParent()) && n.isVisible(),
     );
 
+    const compGroups = useDeepCompareMemoize(
+      applicationGroups.map((g: Node) => ({
+        label: g.getLabel(),
+      })),
+    );
+    const compUnassignedItems = useDeepCompareMemoize(
+      unassignedItems.map((g: Node) => ({
+        label: g.getLabel(),
+      })),
+    );
```

Comment:
Does this work instead? Otherwise use IDs because labels aren't unique.
```suggestion
    const compGroups = React.useMemo(() => applicationGroups, applicationGroups);
    const compUnassignedItems = React.useMemo(() => unassignedItems, unassignedItems);
```

Reply:
I'll change to Ids, useMemo doesn't work.